projects/aca-content/src/lib/components/details/details.component.scss

```diff
@@ -25,14 +35,23 @@ app-details-manager {
 
 .aca-details-tabs {
   margin-top: 40px;
-  height: calc(100% - 80px);
+  height: calc(100% - 136px);
 
   .mat-tab-body-wrapper {
     height: 100%;
   }
 
-  .mat-tab-labels {
-    text-transform: uppercase;
+  .mat-tab-list {
+    margin-left: 50px;
+
+    .mat-tab-labels {
+      .mat-tab-label {
```

Comment:
can we avoid styling mat classes?

Reply:
closing based on https://github.com/Alfresco/alfresco-ng2-components/pull/8995#discussion_r1379736785, please reopen if you think should be fixed